### DISCUSSION TITLE
Is there a setting to prevent tab closing when connection is lost?

### DISCUSSION THREAD

**Discussion by Metal-joker** [Eugeny/tabby#8521] [Q&A]
I just want reconnect to the session to check if my settings correct, but when i type `ctrl+D` or `exit`, tab will auto close

**Comment by lawweiliang** [comment]
@Metal-joker 

Mind to share more information??

Are you talking about SSH connection?

If possible, show us some video or image. 
It would be easier for us to help up yeah.
